dotcom-rendering/src/web/components/DiscussionLayout.tsx

```diff
@@ -0,0 +1,99 @@
+import { css } from '@emotion/react';
+import { from } from '@guardian/source-foundations';
+import { ContainerLayout } from './ContainerLayout';
+import { Flex } from './Flex';
+import { RightColumn } from './RightColumn';
+import { AdSlot } from './AdSlot';
+import { Island } from './Island';
+import { DiscussionContainer } from './DiscussionContainer.importable';
+import { DiscussionMeta } from './DiscussionMeta.importable';
+
+interface DiscussionLayoutProps {
+	format: ArticleFormat;
+	discussionApiUrl: string;
+	shortUrlId: string;
+	discussionD2Uid: string;
+	discussionApiClientHeader: string;
+	enableDiscussionSwitch: boolean;
+	isAdFreeUser: boolean;
+	shouldHideAds: boolean;
+	expanded?: boolean;
```

Comment:
This prop isn't used anywhere. Not sure if it was alway this way but it can be removed

Reply:
In fact, I think it's use further down (in `Discussion`) can be removed. The internal logic of the discussion itself should decide this

Reply:
Ah yeah, worth bearing in mimd!